src/js/components/sharedComponents/commonCards/CardContainer.jsx

```diff
@@ -10,17 +10,29 @@ const propTypes = {
     variant: PropTypes.string, // elevated, outline, or none
     size: PropTypes.string, // sm, md, or lg
     fill: PropTypes.string,
-    height: PropTypes.number
+    height: PropTypes.number,
+    className: PropTypes.string
+};
+
+const flexStyles = {
+    display: "flex",
+    "flex-direction": "column",
+    flex: "1 1 auto",
+    "flex-basis": "auto",
```

Comment:
i think this should be flexBasis

Reply:
![image](https://user-images.githubusercontent.com/102241022/191045237-ba7f802a-3b6c-4ddb-b644-11fd4f3d05ae.png)